x-pack/platform/plugins/shared/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -291,8 +291,8 @@ async function buildDashboardUrlFromSettings(
   // Override with filters and queries from saved dashboard if they are available.
   const { searchSource } = dashboard.attributes.kibanaSavedObjectMeta;
   if (searchSource !== undefined) {
-    if (Array.isArray(searchSource.filter) && searchSource.filter.length > 0) {
-      filters = searchSource.filter;
+    if (Array.isArray(searchSource.filters) && searchSource.filters.length > 0) {
```

Comment:
`filters` is now plural in the dashboard content. I don't think this has any adverse affects in your url editor, but please confirm.